src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -483,6 +483,43 @@ async def _get_alt_ac_start_and_end(
                     return None, error
         return tuple(alt_ac_data_values), None
 
+    async def _get_tx_exons_genomic_coords(
```

Comment:
Naming here seems a little misleading -- I'm mentally expanding it as "get transcript exon's genomic coordinates", but it seems like what we're really doing is "get exon's transcript and genomic coordinates". Can we just call it `_get_exon_coords`?

Reply:
Oh, should we read this as "get transcript's exon's genomic coordinates"? so maybe `_get_tx_exon_coords`? I'm just wondering about including "genomic" in front of "coords" because we fetch coordinates with respect to both the genomic and transcript accession.